app/src/main/java/org/fossasia/openevent/app/common/utils/ui/ViewUtils.java

```diff
@@ -85,4 +86,8 @@ public static void setTitle(Fragment fragment, String title) {
         }
     }
 
+    public static void showCustomSnackbar(View view, String message) {
+        Snackbar.make(view, message, Snackbar.LENGTH_SHORT).show();
```

Comment:
yeah. it would repeat in all the views. also in future snackbar properties can be changed from the single place.